routers/repo/editor.go

```diff
@@ -56,6 +56,12 @@ func getParentTreeFields(treePath string) (treeNames []string, treePaths []strin
 }
 
 func editFile(ctx *context.Context, isNewFile bool) {
+	// Don't allow editing if the repo is archived
```

Comment:
A middleware check isArichived is better I think.

Reply:
Fixed.